.github/workflows/deploy_develop.yml

```
@@ -42,13 +42,13 @@ jobs:
       - name: Cypress run
         uses: cypress-io/github-action@v2
         with:
-          start: yarn serve
-          wait-on: "http://localhost:8080"
+          start: yarn dev
+          wait-on: "http://localhost:3000"
 
       - name: build
         run: |
-          sed -i 's,\(VUE_APP_KEYBOARDS_URL=\).*,\1https://keyboards.develop.qmk.fm,' .env.production
-          echo VUE_APP_TRAVIS_COMMIT=$GITHUB_SHA >> .env.production
+          sed -i 's,\(VITE_APP_KEYBOARDS_URL=\).*,\1https://keyboards.develop.qmk.fm,' .env.production
+          echo VITE_APP_TRAVIS_COMMIT=$GITHUB_SHA >> .env.production
```

Looks to me like this should be

```suggestion
          sed -i 's,\(VITE_KEYBOARDS_URL=\).*,\1https://keyboards.develop.qmk.fm,' .env.production
          echo VITE_TRAVIS_COMMIT=$GITHUB_SHA >> .env.production
```

---

https://vitejs.dev/guide/env-and-mode.html#modes no

---

I guess this whole thing is over my head, because nothing on that page explains to me why not. But I trust your opinion on this more than my own.